doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst

```diff
@@ -0,0 +1,1302 @@
+    * Perform a factory reset of the device and start Bluetooth LE advertising (:kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START`).
+    * Perform a factory reset of the device and then reboot the device (:kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_REBOOT`).
+  * Page about :ref:`ug_matter_ecosystems_certification`.
+  * Page about :ref:`ug_matter_overview_bridge`.
+
+* Updated:
+
+  * Matter over Thread samples so that the OpenThread shell is disabled by default.
+  * The :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option to be enabled by default, including for builds without factory data support.
+    The firmware now erases all flash pages in the non-volatile storage during a factory reset, instead of just clearing Matter-related settings.
+  * The :kconfig:option:`CONFIG_CHIP_EXTENDED_DISCOVERY` Kconfig option to be disabled by default.
+    The commissionable node now does not advertise a commissioning service when it does not have the commissioning window open.
+  * The RAM usage based on test measurements.
+    After the following optimizations, the RAM usage decreased by around 12-20% on all supported boards:
+
+    * Reduced the number of network and Matter stack buffers and packets.
+    * Disabled SSL server support.
+    * Reduced the Main, Matter and OpenThread stack sizes.
+    * Reduced the Mbed TLS heap size.
+    * Improved the buffer usage of the nRF700X driver for Matter.
+    * Reduced the size of the Matter event queue.
+
+  * Page about :ref:`ug_matter_device_certification` with the information about :ref:`ug_matter_device_certification_matter_samples`.
+
+* Fixed:
+
+  * An IPC crash on nRF5340 when Zephyr's main thread takes a long time.
+  * An application core crash on nRF5340 targets with the factory data module enabled.
+    The crash would happen after the OTA firmware update finishes and the image is confirmed.
+
+See `Matter samples`_ for the list of changes for the Matter samples.
+
+Matter fork
++++++++++++
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``v1.1.0.1`` tag.
+
+The following is the most important change inherited from the upstream Matter:
+
+* Added the :kconfig:option:`CONFIG_CHIP_MALLOC_SYS_HEAP_WATERMARKS_SUPPORT` Kconfig option to manage watermark support.
+
+* Updated:
+
+  * The factory data guide with an additional rotating ID information.
+  * Set onboarding code generation to be enabled by default if the :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_BUILD` Kconfig is set.
+
+* Fixed RAM and ROM reports.
+
+Zigbee
+------
+
+The Zigbee stack integrated with the |NCS| is not eligible for certification.
+It should only be used for evaluation and prototyping, and should not be used in an end product.
+A fixed, production ready version of the Zigbee stack will be part of the next release of the |NCS|.
+
```

Comment:
Are you sure we want to add this? We have the following statement in the highlights:

* Deprecated:

  * With the introduction of Matter, all HomeKit customers are recommended to use Matter for new designs of smart home products.
    As a result, HomeKit Accessory Development Kit has been deprecated, and it will be removed in the next release of |NCS|.

Reply:
Added as agreed offline.